templates/activity.mustache

```diff
@@ -101,9 +107,10 @@
 
     <div id="monassis" class="monassis monassis--practice monassis--maths monassis--siyavula-api">
         <div class="question-wrapper">
+            <div id="sequence-num"></div>
             <div class="question-content"></div>
         </div>
     </div>
 </main>
 
-<script src="{{ baseurl }}static/themes/emas/siyavula-api/siyavula-api.min.js"></script>
+<script src="https://thor.siyavula.com/static/themes/emas/siyavula-api/siyavula-api.min.js"></script>
```

Comment:
Remember to change this back.

Reply:
Will do as soon as the review process is done:)